src/components/viewers/MultiFlatmap.vue

```diff
@@ -24,6 +24,35 @@ import EventBus from "../EventBus";
 import store from "../../store";
 import markerZoomLevels from '../markerZoomLevels';
 
+/*
+ * Function to check markers visibility at zoom level.
+ * I have modified it to make sure the marker is displayed
+ * if the uberon is not present in the hardcoded zoom-level list.
+ */
+
+const checkMarkersAtZoomLevel = (flatmapImp, markers, zoomLevel) => {
+  if (markers) {
```

Comment:
This method is only used here and replaces some code where `markers` was always defined, so... probably no need to test `markers` here?

Reply:
The previous code does not check markers. I do not want to make the assumption that markers is defined.

Reply:
Was just trying to be consistent, but agreed.